.github/workflows/markdown-link-check.yml

```diff
@@ -9,13 +9,11 @@ on:
     - main
 
 jobs:
-
-  markdown-link-check:
-    name: Check markdown links
+  linkChecker:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v4
-    - uses: gaurav-nelson/github-action-markdown-link-check@v1
-      with:
-        use-quiet-mode: 'yes'
-        config-file: '.mlc-config.json'
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
```

Comment:
Please add `fail: true` as argument to make the job fail when errors were found, otherwise nobody will realise they have failing links in their PR